style.css

```diff
@@ -418,8 +417,227 @@ button:hover::after {
   background-color: #f27f2d;
 }
 
-@media only screen and (min-width: 768px) {
+@media all and (min-width: 375px) {
+  body {
+    max-width: 375px;
+    margin: auto;
+  }
+}
+
+@media all and (min-width: 768px) {
   .toggler-icon {
     display: none;
   }
+
+  body {
+    max-width: 1440px;
+    height: 870px;
+    margin: auto;
+  }
+
+  .hidden-nav {
+    display: inline;
+  }
+
+  .hidden-nav ul li {
+    margin-left: 16px;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0% 120px;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: row;
+    background: url("./assets/images/deskttopHeaderbg.png");
+    background-color: #1c1a19;
+    background-size: 450px;
+    background-repeat: no-repeat;
+    background-position: 99.9% 0%;
+  }
+
```

Comment:
Kindly note that your project has borders and margins not in the figma design. It is important that your project maches the figma design as much as possible
This is how your project currently is once you close the developer tools:
<img width="1636" alt="Screenshot 2022-09-30 at 22 50 02" src="https://user-images.githubusercontent.com/48546992/193352036-c1e903fe-4735-4514-8bdb-00b703ed9d45.png">

This is how it should be, filling the entire width of device just like your mobile view does:
<img width="717" alt="Screenshot 2022-09-30 at 23 34 43" src="https://user-images.githubusercontent.com/48546992/193352286-be65947d-9a51-404e-85d6-e636ace54c92.png">